### DISCUSSION TITLE
New feature Request: No dimming until X time of day

### DISCUSSION THREAD

**Discussion by GoBooth** [claytonjn/hass-circadian_lighting#171] [Ideas]
I've been using Circadian for over a year now, and this 1 thing has always bugged me.
I absolutely LOVE this integration, I think it's the best part of my smart home.
I love how the Light change from warm to cool back to warm through out the day.

However, during the winter months (at least here in Canada) it gets DARK really early! the sun sets at 6pm (and we're only in November)

My request would be to add an option to NOT dim the lights until X time.

- The available options such as "sunset offset" doesn't really work for me since it's not a static number.
- The "Sunset override" only works for half the year.

To clarify my needs here, I don't want the lights to DIM before 7pm during the winter,  but if it's summer and the sun sets after 7pm, then I want the integration to follow the actual sunset time.

**Comment by jonhamil** [comment]
I've literally just been digging to see how to do this. We have exactly the same issue in the winter in the UK. I wondered if there was another add-on that could trigger the adaptive lighting integration "on" at a certain time and "off" when it's dark outside but realistically too early for the lights to dim indoors.

**Comment by dd1982** [comment]
I live in Sweden so can relate to the issue and was pondering how to solve it. Eventually I decided to simply use fixed times for sunset_time and sunrise_time and have my indoor lighting stay on a fixed schedule regardless of what the sun is doing. Turns out this is a much more acceptable solution than I thought since I realized don't really want the lights to be on full blast late in the evening summertime either. The sun doesn't set until after midnight here. Using a fixed sunset at 20:00 year round works just fine along with some tweaks to the max_color temp such that I can have my lighting fixed at 5K during the day until around 17:00 when circadian lighting is enabled and they slowly starts going warmer from there.
